### ISSUE TITLE
GNOME 47 support

### ISSUE THREAD

**Issue by psydvl** [coadmunkee/gnome-shell-extension-espresso#69]

**Comment by jee-r** [comment]
A Pull Request is already open please thumbs up

https://github.com/coadmunkee/gnome-shell-extension-espresso/pull/68

**Comment by Arcitec** [comment]
Caffeine (the original upstream extension) has modernized it's code to only support new GNOME versions, and has added new features such as Quick Settings integration. It is now the preferred extension again.

Remember that Espresso was forked to delete old GNOME 3 code from the extension. So its purpose is basically not needed anymore.

**Comment by ptzn** [comment]
@Arcitec you are completely right, but there are few features was added into espresso since there (and missed in original caffeine), eg. automatically turning on/off depends on power state.
